### ISSUE TITLE
Small things

### ISSUE THREAD

**Issue by SlimeDog** [EssentialsX/Essentials#2296]
### Information

**Full output of `/ess version`**:

```
[06:45:13 INFO]: CONSOLE issued server command: /ess version
[06:45:13 INFO]: Server version: 1.13.2-R0.1-SNAPSHOT git-Spigot-e769fe4-068dab5 (MC: 1.13.2)
[06:45:13 INFO]: EssentialsX version: dev-45.195
[06:45:13 INFO]: LuckPerms version: 4.3.44
[06:45:13 INFO]: Vault version: 1.7.1-b91
[06:45:13 INFO]: EssentialsXChat version: dev-45.195
[06:45:13 INFO]: You are running an unsupported server version!
```
Note the last line, which is also reported on startup (see below).

### Details

**Description** 
Error message on startup
```
[06:12:43] [Server thread/ERROR]: [Essentials] §6You are running an §4unsupported server version§6.
```
BTW, this message must be hard-coded. The version in my (edited) messages_en.yml does not contain formatting.
 
messages.properties differs (in small ways) from messages_en.properties
```
$ diff messages.properties messages_en.properties 
254c254
< leatherSyntax=\u00a76Leather color syntax\: color\:<red>,<green>,<blue> eg\: color\:255,0,0 OR color\:<rgb int> eg\: color\:16777011
---
> leatherSyntax=\u00a76Leather color syntax\: color\:<red>,<green>,<blue> eg\: color\:255,0,0.
265c265
< mailClear=\u00a76To clear your mail, type\u00a7c /mail clear\u00a76.
---
> mailClear=\u00a76To mark your mail as read, type\u00a7c /mail clear\u00a76.
307c307
< mutedUserSpeaks={0} tried to speak, but is muted: {1}
---
> mutedUserSpeaks={0} tried to speak, but is muted.
470c470
< serverUnsupported=\u00a7cYou are running an unsupported server version!
---
> serverUnsupported=\u00a76You are running an \u00a74unsupported server version\u00a76.
494c494
< sudoExempt=\u00a74You cannot sudo \u00a7c{0}.
---
> sudoExempt=\u00a74You cannot sudo this user.
573,577d572
< versionOutputVaultMissing=\u00a74Vault is not installed. Chat and permissions may not work.
< versionOutputFine=\u00a76{0} version: \u00a7a{1}
< versionOutputWarn=\u00a76{0} version: \u00a7c{1}
< versionOutputUnsupported=\u00a7d{0} \u00a76version: \u00a7d{1}
< versionOutputUnsupportedPlugins=\u00a76You are running \u00a7dunsupported plugins\u00a76!
```

**Comment by SlimeDog** [comment]
How is messages.properties used? I have only .../plugins/Essentials/messages_en.properties installed. Should I have both?

**Comment by mdcfe** [comment]
> ```java
> [06:45:13 INFO]: You are running an unsupported server version!
> ```
> 
> Note the last line, which is also reported on startup (see below).

I've fixed this message appearing erroneously on supported versions in 769dbe885275fc2edb7b0bf17e3959925f6ed2e8.

> messages.properties differs (in small ways) from messages_en.properties

I've addressed all these differences in 67bccb865a8db4bd9e1ff8ee6794dd1321a77ce3 and 48bf264178ef35711b0e90064857d17bf20aba2f.

Thanks for reporting these! The fixes are in the latest [experimental build](https://ci.ender.zone/job/EssentialsX-Experimental).

---

> How is messages.properties used? I have only .../plugins/Essentials/messages_en.properties installed. Should I have both?

`messages.properties` is loaded first and should contain *every* message. `messages_xx_YY.properties` is loaded and any messages in the language-specific file override the ones from `messages.properties`. This means you can create an empty `messages.properties` file and only add the messages you want to override, and EssentialsX will use the defaults for the rest. (I don't know why `messages_en.properties` exists if it's intended to be identical to `messages.properties` ¯\\\_(ツ)\_/¯)
